soroban-react-dapp/contracts/utils/env_config.ts

```diff
@@ -108,8 +112,81 @@ class EnvConfig {
       throw new Error(`${userKey} secret key not found in .env`);
     }
   }
+
+  /**
+   * Creates new accounts
+   * @param count - Number of funded accounts to create
+   */
+  async initializeChildAccounts(count: number = 10): Promise<void> {
+    if (this.childAccounts && this.childAccounts.length > 0) {
+      console.log("Child accounts already exist.");
+      return;
+    }
+
+    this.childAccounts = [];
+    for (let i = 0; i < count; i++) {
+      const pair = Keypair.random();
+      try {
+        const response = await fetch(
+          `https://friendbot.stellar.org?addr=${encodeURIComponent(pair.publicKey())}`,
```

Comment:
This friendbot url only works for testnet, what if i want to use `childAccounts` in standalone/futurenet?

This same file has a `friendbot` that gets loaded when the config gets loaded...

maybe, instead of running `initializeChildAccounts` manually from deploy.ts could be done when the config gets loaded from here

```ts
export const config = (network: string) => {
  return EnvConfig.loadFromFile(network);
};
```

once the config is loaded it knows which is the friendbot_url and could also be from here or loadFromfile where this childAccounts gets created and funded, or maybe just creates them and fund them later as we use them?

Reply:
Yeah but then we will have to await whenever we have to load config. I did like this before itself but chopan preferred not to make loading config async

Reply:
@joaquinsoza I have now changed `initializeChildAccounts` to use url in the friendbot field

Reply:
@RajeshRk18 ohh okay, good to know. checking now